src/app/view/components/listing-data-table/action-header/action-header.module.ts

```diff
@@ -15,9 +16,8 @@ import { ConfirmDialogModule, SplitButtonModule } from 'primeng/primeng';
     imports: [
         CommonModule,
         ActionButtonModule,
-        SplitButtonModule,
-        ConfirmDialogModule
+        SplitButtonModule
     ],
-    providers: []
+    providers: [ DotConfirmationService ]
```

Comment:
By putting this here, you are creating another instance because it's already in the global providers: https://github.com/dotCMS/core-web/blob/8d63b47c5b78d636cb6503ca8225fa03b42a2476/src/app/providers.ts#L40 so you don't need it here.